Updated Author Information and Inclusion of Jupyter Notebooks

nbonfante opened the pull request:
I modified the presentation of the authors by adding their titles,
affiliations, and contact information.
The Jupyter Notebooks, along with their respective scopes, have been added to the Resources section.

nbonfante committed:
Inclusion of Jupyter Notebooks and Their Respective Scopes in Resources

The Jupyter Notebooks, along with their respective scopes, have been added to the Resources section.

sebabedin committed:
Updated Author Information: Titles, Affiliations, and Contact Details

I modified the presentation of the authors by adding their titles,
affiliations, and contact information.

wesleyboar committed:
Merge branch 'main' into main

wesleyboar reviewed (commented):
Hi, thanks for the PR. No [other Use Cases](https://github.com/search?q=repo%3ADesignSafe-CI%2FDS-User-Guide+%22%2F%2F%2F+html+%7C+header%22&type=code) mention titles nor email addresses. The introduction of this data creates inconsistency that gives me pause.

@nbonfante:
1. Is it imperative to list the title and email address of these people?
2. Have these people approved us adding their contact information here?

@silviamazzoni:
1. Should we provide this information?
2. Any preferred format?

**Example (all known data):**

> **[Name (Role)](mailto:someone@example.com) – Lab/Dept, University/Institution**
> ```
> **[Name (Role)](mailto:someone@example.com) – Lab/Dept, University/Institution**
> ```

**Example (minimal data):**

> **Name – University/Institution**
> ```
> **Name – University/Institution**
> ```

wesleyboar commented on user-guide/docs/usecases/pinelli/2usecase.md:
```diff
@@ -6,14 +6,21 @@ Geospatial Hurricane Disaster Reconnaissance Data Integration and Visualization
-**Ajaz, M.A. – Florida Institute of Technology**   
+**Pinelli, J-P. – Professor - Florida Institute of Technology - pinelli@fit.edu**    <br> 
+**Sziklay, E. – Graduate Student - Florida Institute of Technology - esziklay2018@my.fit.edu**  <br> 
+**Ajaz, M.A. – Graduate Student - Florida Institute of Technology - abdulajaz2023@my.fit.edu**   
```
```suggestion
**Pinelli, J-P. – Professor - Florida Institute of Technology**<br> 
**Sziklay, E. – Graduate Student - Florida Institute of Technology**<br> 
**Ajaz, M.A. – Graduate Student - Florida Institute of Technology**

```

wesleyboar reviewed (changes requested):
After internal discussion [regarding author data](https://github.com/DesignSafe-CI/DS-User-Guide/pull/111#pullrequestreview-2565658945), we've opted **not** to add contact information.[^1]

[^1]: [another instance of declining this, and our reasons](https://github.com/DesignSafe-CI/DS-User-Guide/pull/113#pullrequestreview-2565668410)

wesleyboar committed:
fix: remove contact info

reasons: https://github.com/DesignSafe-CI/DS-User-Guide/pull/113#pullrequestreview-2565668410
